project.mml

```diff
@@ -1992,7 +1992,8 @@ Layer:
             name,
             CASE WHEN building = 'no' OR building IS NULL THEN 'no' ELSE 'yes' END AS is_building -- always no with the where conditions
           FROM planet_osm_polygon
-          WHERE (landuse IN ('forest', 'military', 'farmland')
+          WHERE way && !bbox!
+            AND (landuse IN ('forest', 'military', 'farmland')
```

Comment:
AND has higher precedence than OR In SQL, so we need a pair or brackers around all the OR statements.

Reply:
The brackets look good to me as-is.